src/Character.js

```diff
@@ -47,10 +55,25 @@ export default class Character extends React.Component<CharacterProps, {}> {
     render() {
         return (
             <g
+                ref={this.characterRef}
                 className='Character'
                 transform={this.props.transform}>
                 {this.getThemedCharacter()}
             </g>
         );
     }
+
+    componentDidMount() {
+        if (this.characterRef.current.scrollIntoView) {
+            this.characterRef.current.scrollIntoView({ behavior: 'auto', block: 'nearest', inline: 'nearest' });
```

Comment:
If we never use different options, we should find a way to reuse them cleanly, so that they can't get out of sync if we need to change anything.  Defining a constant or creating a small helper function would be fine.

Reply:
This change was introduced in 286, so we should resolve any issue to this in 286; however, in Character component, the scrollIntoView property object is used in two places, and they have different scopes, so we can't have a variable for the object, unless we specify that in a class property.

Reply:
Thanks for the explanation.  Given that the Character only uses the function in a single way, a class property would seem to make sense.